sdcm/sla/libs/sla_utils.py

```diff
@@ -375,3 +385,31 @@ def clean_auth(entities_list_of_dict):
                         auth.drop()
                     except Exception as error:  # pylint: disable=broad-except
                         LOGGER.error("Failed to drop '%s'. Error: %s", auth.name, error)
+
+    @staticmethod
+    @retrying(n=100, sleep_time=3, message="Wait for service level has been propagated to all nodes",
```

Comment:
I am checking how long it may take and will open the issue

Reply:
@roydahan 
For now I see the SL + scheduler group are created maximum in 11 sec. Or scheduler group does not create at all, and we have ans issue for that. 
So I decrease waiting time to 120 sec, add adaptive_timeout for 15 sec to measure how long time it will take. 
We do not need to open the issue now